projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,41 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=harbor
+REPO_OWNER=goharbor
+REPO_SUBPATH=./src
+
+DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
+
+FETCH_BINARIES_TARGETS=eksa/helm/chartmuseum
+
+IMAGE_NAMES=harbor-chartserver
+
+PRECOMPILE_PREPARE=swagger_prepare
+
+include $(BASE_DIRECTORY)/Common.mk
+
+HARBOR_CHARTSERVER_IMAGE_COMPONENT=harbor/harbor-chartserver
+
+$(call IMAGE_TARGETS_FOR_NAME, harbor-chartserver): harbor-chartserver-useradd/images/export
+
+harbor-chartserver-useradd/images/export: IMAGE_USERADD_USER_NAME=chart IMAGE_USERADD_USER_ID=10000
+harbor-chartserver-useradd/images/export: scripts_prepare
+
+HARBOR_BASE_NAMESPACE=goharbor
+IMAGE_BUILD_ARGS=HARBOR_BASE_NAMESPACE
+
+swagger_prepare:
+	make -C $(REPO) gen_apis
+
+scripts_prepare:
```

Comment:
Lets make this a real path target.  What we using do is define it as a var up top kind of like this

```
HARBOR_SCRIPTS_TARGET=$(OUTPUT_DIR)/make/photon/common/install_cert.sh
....
$(HARBOR_SCRIPTS_TARGET):
  
  
...


```